persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/PersistentEntity.scala

```diff
@@ -17,9 +18,11 @@ object PersistentEntity {
    *
    * `akka.Done` can optionally be used as a "standard" acknowledgment message.
    *
-   * Type parameter `R` is the type of the reply message.
+   * @tparam R the type of the reply message
    */
-  trait ReplyType[-R]
+  trait ReplyType[R] {
+    type ReplyType = R
```

Comment:
Can you show an example where it's a problem?
In `TestEntity` we have several commands with reply type `Evt` and replying with concrete events such as `Appended`.